### PR TITLE
Added GSoD 21 proposal blog : Shraddha and Soumi 

### PR DESCRIPTION
**Pull request by Soumi7**
Added the combined proposal blog post for GSoD 21 project References section by @shraddhavp and @Soumi7 .

### CODE REVIEW

File: jekyll/_posts/2021-05-11-shraddhavp-soumi-combined-reference-proposal.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+* The references page is difficult to understand for first time users technically.
+
+* It is incomplete and abstract.
+
+* Many information is given in Chinese language and hence difficult for users to understand.
+
+## Proposal Video
+
+<iframe width="1028" height="578" src="https://www.youtube.com/embed/82rFox9Vq14" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (huan) [inline jekyll/_posts/2021-05-11-shraddhavp-soumi-combined-reference-proposal.md]: Please use the following code to simplify your YouTube video embedding:

```html
{% include iframe.html src="https://www.youtube.com/embed/82rFox9Vq14" %}
```

See: https://wechaty.js.org/2020/08/24/add-video-to-wechaty-blog/

@wechaty/writer

**Review by huan — CHANGES_REQUESTED** [review]

**Comment by shraddhavp** [comment]
have added the changes as requested @huan

**Review by huan — APPROVED** [review]
LGTM

**Comment by huan** [comment]
Thank you very much for submitting our first team proposal blog with your great presentation video!

File: jekyll/_posts/2021-05-11-shraddhavp-soumi-combined-reference-proposal.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+title: "2021 Google Season of Docs Project - References"
+author: soumi7
+categories: proposal
+tags:
+  - documentation
+  - gsod
+  - 2021
+  - project
+  - proposal
+image: /assets/2021/05-shraddhavp-soumi-combined-reference-proposal/gsod_s.png
```
Comment (huan) [inline jekyll/_posts/2021-05-11-shraddhavp-soumi-combined-reference-proposal.md:11]: Could we please replace this teaser image with our beautiful first page in the presentation?

image

Comment (shraddhavp) [reply jekyll/_posts/2021-05-11-shraddhavp-soumi-combined-reference-proposal.md:11]: @huan you mean that image must be replaced with the below one ?
firstpage

Comment (shraddhavp) [reply jekyll/_posts/2021-05-11-shraddhavp-soumi-combined-reference-proposal.md:11]: @huan  you mean that image must be replaced with the below one 
Screenshot (154)

Comment (huan) [reply jekyll/_posts/2021-05-11-shraddhavp-soumi-combined-reference-proposal.md:11]: The current version of our post will use `/assets/2021/05-shraddhavp-soumi-combined-reference-proposal/gsod_s.png` as the teaser image.

In our blog post list page and the blog post page, the teaser image will show as the following screenshots:

image

image

The above image is too general and can be improved by replacing it with a more meaningful image.

My suggestion is to replace the current teaser image with  a beautiful and meanful image from your slide, for example, the first page or the second page from the slide will all be good.

Please feel free to ask me if I explains not clear enough, thanks!

Comment (shraddhavp) [reply jekyll/_posts/2021-05-11-shraddhavp-soumi-combined-reference-proposal.md:11]: ya sure @huan

**Pull request merged**